ReleaseConfig

```diff
@@ -37,7 +37,7 @@ version_tuple = (4, 8, 2, 'a', 0)
 # when that version is used.  Python versions prior to deprecate_python_version
 # cause a warning to be issued (assuming it's not disabled).  These values are
 # mandatory and must be present in the configuration file.
-unsupported_python_version = (3, 6, 0)
+unsupported_python_version = (3, 7, 0)
```

Comment:
made a previous comment here which doesn't align with waiting to deprecate 3.7 - removed.